### PR TITLE
Add ARM64 build

### PR DESCRIPTION
**Pull request by dszmigielski**
## Why

Fixes #

## Checklist

- [ ] `CHANGELOG.md` is updated.
- [ ] Documentation is updated.
- [ ] New features are covered by tests.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -89,6 +89,39 @@ jobs:
           git diff
           [[ -z "$(git status --porcelain)" ]]
 
+  build-arm64:
+    strategy:
+      fail-fast: false
+    runs-on: ubuntu-20.04
```
Comment (RassK) [inline .github/workflows/ci.yml:95]: There are no arm machines on the list, all available machines are x64 based. 
I think GH only supports arm on self hosted runners.

Comment (dszmigielski) [reply .github/workflows/ci.yml:95]: I was trying to work around that with uraimo/run-on-arch-action@v2.5.0, but I cannot get it working, so I assume we have to wait for gh actions to add support.

Comment (RassK) [reply .github/workflows/ci.yml:95]: Ah I see, it's using QEMU emulator. Might be easier to test that locally first.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -89,6 +89,39 @@ jobs:
...
+        run: |
+          touch /etc/apt/sources.list.d/backports.list
+          echo "deb http://deb.debian.org/debian stretch-backports main" >> /etc/apt/sources.list.d/backports.list
+          apt-get update
+          apt-get upgrade
+          apt-get install -y curl clang cmake make protobuf-compiler
+          apt-get install -y -t stretch-backports libgrpc++-dev libgrpc++1 libgrpc6 libgrpc-dev protobuf-compiler-grpc 
+          curl -sSL https://dot.net/v1/dotnet-install.sh --output dotnet-install.sh
+          chmod +x ./dotnet-install.sh
+          ./dotnet-install.sh -c 6.0 --install-dir /usr/share/dotnet --no-path
```
Comment (RassK) [inline .github/workflows/ci.yml:113]:
```suggestion
          ./dotnet-install.sh -channel 6.0.xxx --install-dir /usr/share/dotnet
```

Could try this one. Seems it doesn't know about the installed dotnet versions.

Comment (dszmigielski) [reply .github/workflows/ci.yml:113]: Will do, cheers :)

**Comment by Kielek** [comment]
Closing for now. Not ARM64 env. available.

**Pull request closed without merging**